draft-vvv-webtransport-quic.md

```diff
@@ -233,6 +233,42 @@ datagram ID SHALL be absent.
 
 The datagrams sent using QuicTransport MUST be subject to congestion control.
 
+# QuicTransport URI Scheme {#uri}
+
+NOTE: the URI scheme defintion in this section is provisional and subject to
+change, especially the name of the scheme.
+
+QuicTransport uses `quic-transport` URI scheme for identifying QuicTransport
+servers.
+
+The syntax definition below uses Augmented Backus-Naur Form (ABNF) {{!RFC5234}}.
+The definitions of `host`, `port`, `path-abempty`, `query` and `fragment` are
```

Comment:
path-abempty is never used in the ABNF below.

Reply:
Fixed.